### ISSUE TITLE
lsp-plugins-comp-delay-mono v1.2.11 segfaults on start

### ISSUE THREAD

**Issue by LebedevRI** [sadko4u/lsp-plugins#348]
This is on debian unstable, using pipewire
```
(gdb) r
Starting program: /usr/bin/lsp-plugins-comp-delay-mono 
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
Trying to connect to JACK
[New Thread 0x7ffff59ff6c0 (LWP 36265)]
[Thread 0x7ffff59ff6c0 (LWP 36265) exited]
[New Thread 0x7ffff59ff6c0 (LWP 36266)]
[New Thread 0x7ffff51fe6c0 (LWP 36267)]
Successfully connected to JACK

Thread 4 "lsp-plugins-com" received signal SIGSEGV, Segmentation fault.
[Switching to Thread 0x7ffff51fe6c0 (LWP 36267)]
0x00007ffff74bcf5e in ?? () from /usr/lib/lsp-plugins/liblsp-plugins-jack-1.2.11.so
(gdb) Quitad a
(gdb) bt full
#0  0x00007ffff74bcf5e in ?? () from /usr/lib/lsp-plugins/liblsp-plugins-jack-1.2.11.so
No symbol table info available.
#1  0x00007ffff51fdac0 in ?? ()
No symbol table info available.
#2  0x0000000000002000 in ?? ()
No symbol table info available.
#3  0x000055555556b460 in ?? ()
No symbol table info available.
#4  0x00007ffff51fdac0 in ?? ()
No symbol table info available.
#5  0x0000000000000002 in ?? ()
No symbol table info available.
#6  0x00007ffff74bcfca in ?? () from /usr/lib/lsp-plugins/liblsp-plugins-jack-1.2.11.so
No symbol table info available.
#7  0x0000037f00000002 in ?? ()
No symbol table info available.
#8  0x0000000000001fa4 in ?? ()
No symbol table info available.
#9  0x0000000000004e92 in ?? ()
No symbol table info available.
#10 0x00000000347014b7 in ?? ()
No symbol table info available.
#11 0x00007ffff7f96000 in ?? ()
No symbol table info available.
#12 0xb07b3b1e90bc6300 in ?? ()
No symbol table info available.
#13 0x00007ffff49fe000 in ?? ()
No symbol table info available.
#14 0x0000555555c3ce50 in ?? ()
No symbol table info available.
#15 0x0000555555c3ce50 in ?? ()
No symbol table info available.
#16 0xb07b3b1e90bc6300 in ?? ()
No symbol table info available.
#17 0x0000555555c3ce50 in ?? ()
No symbol table info available.
#18 0x0000000000002000 in ?? ()
No symbol table info available.
#19 0x0000555555c3d388 in ?? ()
No symbol table info available.
#20 0x00007ffff7a082d6 in ?? () from /usr/lib/x86_64-linux-gnu/pipewire-0.3/jack/libjack.so.0
No symbol table info available.
#21 0x00007ffff65e6bce in ?? () from /usr/lib/x86_64-linux-gnu/spa-0.2/support/libspa-support.so
No symbol table info available.
#22 0x00007ffff6cc4f90 in ?? () from /lib/x86_64-linux-gnu/libpipewire-0.3.so.0
No symbol table info available.
#23 0x00007ffff7aa63ec in start_thread (arg=<optimized out>) at ./nptl/pthread_create.c:444
        ret = <optimized out>
        pd = <optimized out>
        out = <optimized out>
        unwind_buf = {cancel_jmp_buf = {{jmp_buf = {140737348526384, 3050228396514902818, -152, 2, 140737488344224, 140737297506304, -3050207229810277598, -3050210439459994846}, mask_was_saved = 0}}, priv = {pad = {0x0, 0x0, 0x0, 0x0}, data = {prev = 0x0, 
              cleanup = 0x0, canceltype = 0}}}
        not_first_call = <optimized out>
#24 0x00007ffff7b26a4c in clone3 () at ../sysdeps/unix/sysv/linux/x86_64/clone3.S:81
No locals.
(gdb) exit
A debugging session is active.

        Inferior 1 [process 36255] will be killed.

Quit anyway? (y or n) t
Please answer y or n.
A debugging session is active.

        Inferior 1 [process 36255] will be killed.

Quit anyway? (y or n) y
```
Have anyone else seen this? This could be a problem on pipewire's side.

**Comment by sadko4u** [comment]
Is it possible for you to build plugins in debug mode and reproduce the case in GDB?

**Comment by LebedevRI** [comment]
Here's with debug symbols installed, is this better?
```
(gdb) r
Starting program: /usr/bin/lsp-plugins-comp-delay-mono 
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
Trying to connect to JACK
[New Thread 0x7ffff59ff6c0 (LWP 38386)]
[Thread 0x7ffff59ff6c0 (LWP 38386) exited]
[New Thread 0x7ffff59ff6c0 (LWP 38387)]
[New Thread 0x7ffff51fe6c0 (LWP 38388)]
Successfully connected to JACK

Thread 4 "lsp-plugins-com" received signal SIGSEGV, Segmentation fault.
[Switching to Thread 0x7ffff51fe6c0 (LWP 38388)]
0x00007ffff74bcf5e in lsp::jack::Wrapper::run (this=this@entry=0x55555556b460, samples=samples@entry=8192) at ./modules/lsp-plugin-fw/include/lsp-plug.in/plug-fw/wrap/jack/impl/wrapper.h:386
386     ./modules/lsp-plugin-fw/include/lsp-plug.in/plug-fw/wrap/jack/impl/wrapper.h: No such file or directory.
(gdb) thread apply all bt full

Thread 4 (Thread 0x7ffff51fe6c0 (LWP 38388) "lsp-plugins-com"):
#0  0x00007ffff74bcf5e in lsp::jack::Wrapper::run (this=this@entry=0x55555556b460, samples=samples@entry=8192) at ./modules/lsp-plugin-fw/include/lsp-plug.in/plug-fw/wrap/jack/impl/wrapper.h:386
        port = 0x555555572570
        i = 0
        n = 17
        ui_active = <optimized out>
        dump_req = <optimized out>
        latency = <optimized out>
#1  0x00007ffff74bcfca in lsp::jack::Wrapper::process (nframes=8192, arg=0x55555556b460) at ./modules/lsp-plugin-fw/include/lsp-plug.in/plug-fw/wrap/jack/impl/wrapper.h:688
        ctx = {top = 2, data = {895, 8100, 0, 21600, 0, 867451358, 0, 4160315392, 32767, 819524352, 4003368271, 4104118272, 32767, 1438894800, 21845}}
        result = <optimized out>
        _this = 0x55555556b460
#2  0x00007ffff7a082d6 in on_rtsocket_condition (data=0x555555c3cad0, fd=<optimized out>, mask=<optimized out>) at ../pipewire-jack/src/pipewire-jack.c:1879
        res = 0
        buffer_frames = 8192
        status = 0
        c = 0x555555c3cad0
        __func__ = "on_rtsocket_condition"
#3  0x00007ffff65e6bce in ?? () from /usr/lib/x86_64-linux-gnu/spa-0.2/support/libspa-support.so
No symbol table info available.
#4  0x00007ffff6cc4f90 in ?? () from /lib/x86_64-linux-gnu/libpipewire-0.3.so.0
No symbol table info available.
#5  0x00007ffff7aa63ec in start_thread (arg=<optimized out>) at ./nptl/pthread_create.c:444
        ret = <optimized out>
        pd = <optimized out>
        out = <optimized out>
        unwind_buf = {cancel_jmp_buf = {{jmp_buf = {140737348526384, -4025945552524454153, -152, 2, 140737488344224, 140737297506304, 4025969465798121207, 4025963587698644727}, mask_was_saved = 0}}, priv = {pad = {0x0, 0x0, 0x0, 0x0}, data = {prev = 0x0, cleanup = 0x0, canceltype = 0}}}
        not_first_call = <optimized out>
#6  0x00007ffff7b26a4c in clone3 () at ../sysdeps/unix/sysv/linux/x86_64/clone3.S:81
No locals.

Thread 3 (Thread 0x7ffff59ff6c0 (LWP 38387) "pw-comp_delay_m"):
#0  0x00007ffff7b26e56 in epoll_wait (epfd=4, events=0x7ffff59feab0, maxevents=32, timeout=-1) at ../sysdeps/unix/sysv/linux/epoll_wait.c:30
        sc_ret = -4
        sc_cancel_oldtype = 0
        sc_ret = <optimized out>
#1  0x00007ffff65f57b8 in ?? () from /usr/lib/x86_64-linux-gnu/spa-0.2/support/libspa-support.so
No symbol table info available.
#2  0x00007ffff65e6d3d in ?? () from /usr/lib/x86_64-linux-gnu/spa-0.2/support/libspa-support.so
No symbol table info available.
#3  0x00007ffff6d26f65 in ?? () from /lib/x86_64-linux-gnu/libpipewire-0.3.so.0
No symbol table info available.
#4  0x00007ffff7aa63ec in start_thread (arg=<optimized out>) at ./nptl/pthread_create.c:444
        ret = <optimized out>
        pd = <optimized out>
        out = <optimized out>
        unwind_buf = {cancel_jmp_buf = {{jmp_buf = {140737348526384, -4025945552524454153, -152, 2, 140737488344096, 140737305899008, 4025968367897106167, 4025963587698644727}, mask_was_saved = 0}}, priv = {pad = {0x0, 0x0, 0x0, 0x0}, data = {prev = 0x0, cleanup = 0x0, canceltype = 0}}}
        not_first_call = <optimized out>
#5  0x00007ffff7b26a4c in clone3 () at ../sysdeps/unix/sysv/linux/x86_64/clone3.S:81
No locals.

Thread 1 (Thread 0x7ffff7e92440 (LWP 38383) "lsp-plugins-com"):
#0  0x00007ffff74bee97 in lsp::jack::UIWrapper::sync (this=this@entry=0x55555556fbd0, ts=ts@entry=1697654516418) at ./modules/lsp-plugin-fw/include/lsp-plug.in/plug-fw/wrap/jack/impl/ui_wrapper.h:382
        ctx = {top = 2, data = {895, 8100, 395, 4294957312, 32767, 4294957312, 32767, 4294957312, 32767, 4294957328, 32767, 4152786176, 32767, 4155427097, 32767}}
        pos = <optimized out>
        sync = <optimized out>
        kvt = <optimized out>
        sp = <optimized out>
#1  0x00007ffff74c0503 in lsp::jack::sync_state (arg=0x7ffff7867d00 <lsp::jack::wrapper>, ctime=1697654516418, sched=1697654516418) at wrap/jack.cpp:717
--Type <RET> for more, q to quit, c to continue without paging--c
        w = 0x7ffff7867d00 <lsp::jack::wrapper>
        jw = 0x55555556b460
        uw = 0x55555556fbd0
        w = <optimized out>
        jw = <optimized out>
        uw = <optimized out>
#2  lsp::jack::plugin_main (w=w@entry=0x7ffff7867d00 <lsp::jack::wrapper>) at wrap/jack.cpp:751
        delay = <optimized out>
        res = 0
        period = 40
        ctime = {seconds = 1697654516, nanos = 418428082}
        ts1 = 1697654516418
        ts2 = <optimized out>
#3  0x00007ffff74c0940 in plug_fw_jack_main (plugin_id=<optimized out>, argc=<optimized out>, argv=<optimized out>) at wrap/jack.cpp:872
        res = <optimized out>
        cmdline = {cfg_file = 0x0, plugin_id = 0x555555556069 "comp_delay_mono", parent_id = 0x0, headless = false, list = false, version = false, routing = {v = {nItems = 0, vItems = 0x0, nCapacity = 0, nSizeOf = 16}}}
        lsp_finally_function3__ = {sActor = {__cmdline = @0x7fffffffd9b0}}
        w = <optimized out>
        lsp_finally_function4__ = <optimized out>
#4  0x000055555555528d in main (argc=1, argv=0x7fffffffdb68) at ./modules/lsp-plugin-fw/include/lsp-plug.in/plug-fw/wrap/jack/main/posix.h:404
        hInstance = 0x55555557b450
        version = <optimized out>
        jack_main = <optimized out>
        code = <optimized out>
(gdb) exit
A debugging session is active.

        Inferior 1 [process 38383] will be killed.

Quit anyway? (y or n) y

```

**Comment by LebedevRI** [comment]
Here's with debug symbols installed, is this better?
```
(gdb) r
Starting program: /usr/bin/lsp-plugins-comp-delay-mono 
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
Trying to connect to JACK
[New Thread 0x7ffff59ff6c0 (LWP 38647)]
[Thread 0x7ffff59ff6c0 (LWP 38647) exited]
[New Thread 0x7ffff59ff6c0 (LWP 38648)]
[New Thread 0x7ffff51fe6c0 (LWP 38649)]
Successfully connected to JACK

Thread 4 "lsp-plugins-com" received signal SIGSEGV, Segmentation fault.
[Switching to Thread 0x7ffff51fe6c0 (LWP 38649)]
0x00007ffff74bcf5e in lsp::jack::Wrapper::run (this=this@entry=0x55555556b460, samples=samples@entry=8192) at ./modules/lsp-plugin-fw/include/lsp-plug.in/plug-fw/wrap/jack/impl/wrapper.h:386
386     ./modules/lsp-plugin-fw/include/lsp-plug.in/plug-fw/wrap/jack/impl/wrapper.h: No such file or directory.
(gdb) thread apply all bt full

Thread 4 (Thread 0x7ffff51fe6c0 (LWP 38649) "lsp-plugins-com"):
#0  0x00007ffff74bcf5e in lsp::jack::Wrapper::run (this=this@entry=0x55555556b460, samples=samples@entry=8192) at ./modules/lsp-plugin-fw/include/lsp-plug.in/plug-fw/wrap/jack/impl/wrapper.h:386
        port = 0x555555572570
        i = 0
        n = 17
        ui_active = <optimized out>
        dump_req = <optimized out>
        latency = <optimized out>
#1  0x00007ffff74bcfca in lsp::jack::Wrapper::process (nframes=8192, arg=0x55555556b460) at ./modules/lsp-plugin-fw/include/lsp-plug.in/plug-fw/wrap/jack/impl/wrapper.h:688
        ctx = {top = 2, data = {895, 8100, 0, 21717, 0, 886748447, 0, 4160315392, 32767, 2193931520, 2994903878, 4104118272, 32767, 1438893792, 21845}}
        result = <optimized out>
        _this = 0x55555556b460
#2  0x00007ffff7a082d6 in on_rtsocket_condition (data=0x555555c3c6e0, fd=<optimized out>, mask=<optimized out>) at ../pipewire-jack/src/pipewire-jack.c:1879
        res = 0
        buffer_frames = 8192
        status = 0
        c = 0x555555c3c6e0
        __func__ = "on_rtsocket_condition"
#3  0x00007ffff65e6bce in loop_iterate_cancel (object=object@entry=0x555555c61d28, timeout=timeout@entry=-1) at ../spa/plugins/support/loop.c:454
        s = <optimized out>
        __cancel_buf = {__cancel_jmp_buf = {{__cancel_jmp_buf = {93824999628240, 3027226915665978023, 4294967295, 2, 140737488344224, 140737297506304, 9175978021119959719, 9175971986107900583}, __mask_was_saved = 0}}, __pad = {0x7ffff51fde60, 0x0, 0x0, 0x0}}
        __cancel_routine = 0x7ffff65e43c0 <cancellation_handler>
        __cancel_arg = <optimized out>
        __not_first_call = <optimized out>
        impl = 0x555555c61d28
        ep = {{events = 1, data = 0x7ffff004ef90}, {events = 0, data = 0x0} <repeats 31 times>}
        e = <optimized out>
        i = 0
        nfds = 1
        cdata = {ep = 0x7ffff51fdc10, ep_count = 1}
        __func__ = "loop_iterate_cancel"
#4  0x00007ffff6cc4f90 in do_loop (user_data=0x555555c401d0) at ../src/pipewire/data-loop.c:65
        __cancel_buf = {__cancel_jmp_buf = {{__cancel_jmp_buf = {93824999628072, -9175955205028207961, -152, 2, 140737488344224, 140737297506304, 9175978020956381863, 9175970732965208743}, __mask_was_saved = 0}}, __pad = {0x7ffff51fdef0, 0x0, 0x7ffff65cf5b9, 0x7ffff51fecdc}}
        __cancel_routine = 0x7ffff6cc4db0 <thread_cleanup>
        __cancel_arg = 0x555555c401d0
        __not_first_call = <optimized out>
        this = <optimized out>
        res = <optimized out>
        cb = <optimized out>
        m = 0x4000
        data = <optimized out>
        iterate = <optimized out>
        __func__ = "do_loop"
#5  0x00007ffff7aa63ec in start_thread (arg=<optimized out>) at ./nptl/pthread_create.c:444
        ret = <optimized out>
        pd = <optimized out>
        out = <optimized out>
        unwind_buf = {cancel_jmp_buf = {{jmp_buf = {140737348526384, -9175955205028207961, -152, 2, 140737488344224, 140737297506304, 9175978020935410343, 9175972158138852007}, mask_was_saved = 0}}, priv = {pad = {0x0, 0x0, 0x0, 0x0}, data = {prev = 0x0, cleanup = 0x0, canceltype = 0}}}
        not_first_call = <optimized out>
#6  0x00007ffff7b26a4c in clone3 () at ../sysdeps/unix/sysv/linux/x86_64/clone3.S:81
No locals.

Thread 3 (Thread 0x7ffff59ff6c0 (LWP 38648) "pw-comp_delay_m"):
#0  0x00007ffff7b26e56 in epoll_wait (epfd=4, events=events@entry=0x7ffff59feab0, maxevents=32, timeout=-1) at ../sysdeps/unix/sysv/linux/epoll_wait.c:30
        sc_ret = -4
        sc_cancel_oldtype = 0
        sc_ret = <optimized out>
#1  0x00007ffff65f57b8 in impl_pollfd_wait (object=<optimized out>, pfd=<optimized out>, ev=0x7ffff59fec80, n_ev=<optimized out>, timeout=<optimized out>) at ../spa/plugins/support/system.c:137
        ep = 0x7ffff59feab0
--Type <RET> for more, q to quit, c to continue without paging--c
        i = <optimized out>
        nfds = <optimized out>
#2  0x00007ffff65e6d3d in loop_iterate (object=0x555555c59bb8, timeout=-1) at ../spa/plugins/support/loop.c:471
        _f = <optimized out>
        _res = -95
        _o = <optimized out>
        impl = 0x555555c59bb8
        ep = {{events = 1, data = 0x555555c72090}, {events = 0, data = 0x0}, {events = 0, data = 0x7ffff7eace90 <_Unwind_ForcedUnwind>}, {events = 0, data = 0x0}, {events = 0, data = 0x0}, {events = 0, data = 0x0}, {events = 4120899032, data = 0x7ffff59fede0}, {events = 0, data = 0x7ffff59feed0}, {events = 0, data = 0x0}, {events = 4120899088, data = 0x0}, {events = 0, data = 0x0}, {events = 0, data = 0x0}, {events = 4120899056, data = 0x7ffff59fedf8}, {events = 4120899072, data = 0x7ffff59fee08}, {events = 4120899288, data = 0x0}, {events = 4120899296, data = 0x7ffff7aa63ec <start_thread+700>}, {events = 0, data = 0x0}, {events = 0, data = 0x7ffff7aa6130 <start_thread>}, {events = 0, data = 0x0}, {events = 0, data = 0x0}, {events = 0, data = 0x0}, {events = 4155172240, data = 0x7ffff7eace90 <_Unwind_ForcedUnwind>}, {events = 4120899312, data = 0x555555c61d28}, {events = 4294967144, data = 0x2}, {events = 4294955936, data = 0x7ffff51ff000}, {events = 0, data = 0x7ffff7aae710 <__GI___pthread_unwind+64>}, {events = 4112510976, data = 0x7ffff7aae72d}, {events = 1439046952, data = 0x7ffff6cc50dc <do_loop+540>}, {events = 0, data = 0x555555c401d0}, {events = 1439046952, data = 0x7ffff65e69d0 <loop_iterate_cancel>}, {events = 1439046952, data = 0x80a8755b7965a2a7}, {events = 4294967144, data = 0x2}}
        e = <optimized out>
        i = <optimized out>
        nfds = <optimized out>
#3  0x00007ffff6d26f65 in do_loop (user_data=0x555555c3cf60) at ../src/pipewire/thread-loop.c:295
        _f = <optimized out>
        _res = <optimized out>
        _o = <optimized out>
        this = 0x555555c3cf60
        res = <optimized out>
        __func__ = "do_loop"
#4  0x00007ffff7aa63ec in start_thread (arg=<optimized out>) at ./nptl/pthread_create.c:444
        ret = <optimized out>
        pd = <optimized out>
        out = <optimized out>
        unwind_buf = {cancel_jmp_buf = {{jmp_buf = {140737348526384, -9175955205028207961, -152, 2, 140737488344096, 140737305899008, 9175976920886911655, 9175972158138852007}, mask_was_saved = 0}}, priv = {pad = {0x0, 0x0, 0x0, 0x0}, data = {prev = 0x0, cleanup = 0x0, canceltype = 0}}}
        not_first_call = <optimized out>
#5  0x00007ffff7b26a4c in clone3 () at ../sysdeps/unix/sysv/linux/x86_64/clone3.S:81
No locals.

Thread 1 (Thread 0x7ffff7e92440 (LWP 38644) "lsp-plugins-com"):
#0  0x00007ffff74bee97 in lsp::jack::UIWrapper::sync (this=this@entry=0x55555556fbd0, ts=ts@entry=1697654633431) at ./modules/lsp-plugin-fw/include/lsp-plug.in/plug-fw/wrap/jack/impl/ui_wrapper.h:382
        ctx = {top = 2, data = {895, 8100, 395, 4294957312, 32767, 4294957312, 32767, 4294957312, 32767, 4294957328, 32767, 4152786176, 32767, 4155427097, 32767}}
        pos = <optimized out>
        sync = <optimized out>
        kvt = <optimized out>
        sp = <optimized out>
#1  0x00007ffff74c0503 in lsp::jack::sync_state (arg=0x7ffff7867d00 <lsp::jack::wrapper>, ctime=1697654633431, sched=1697654633431) at wrap/jack.cpp:717
        w = 0x7ffff7867d00 <lsp::jack::wrapper>
        jw = 0x55555556b460
        uw = 0x55555556fbd0
        w = <optimized out>
        jw = <optimized out>
        uw = <optimized out>
#2  lsp::jack::plugin_main (w=w@entry=0x7ffff7867d00 <lsp::jack::wrapper>) at wrap/jack.cpp:751
        delay = <optimized out>
        res = 0
        period = 40
        ctime = {seconds = 1697654633, nanos = 431121299}
        ts1 = 1697654633431
        ts2 = <optimized out>
#3  0x00007ffff74c0940 in plug_fw_jack_main (plugin_id=<optimized out>, argc=<optimized out>, argv=<optimized out>) at wrap/jack.cpp:872
        res = <optimized out>
        cmdline = {cfg_file = 0x0, plugin_id = 0x555555556069 "comp_delay_mono", parent_id = 0x0, headless = false, list = false, version = false, routing = {v = {nItems = 0, vItems = 0x0, nCapacity = 0, nSizeOf = 16}}}
        lsp_finally_function3__ = {sActor = {__cmdline = @0x7fffffffd9b0}}
        w = <optimized out>
        lsp_finally_function4__ = <optimized out>
#4  0x000055555555528d in main (argc=1, argv=0x7fffffffdb68) at ./modules/lsp-plugin-fw/include/lsp-plug.in/plug-fw/wrap/jack/main/posix.h:404
        hInstance = 0x55555557b450
        version = <optimized out>
        jack_main = <optimized out>
        code = <optimized out>
(gdb) exit
```

**Comment by sadko4u** [comment]
No, it doesn't give the exact crash point.

**Comment by LebedevRI** [comment]
Sorry, i have never manually built LSP, and not sure i'm motivated to deal with this :/
I take it you are unable to reproduce?

**Comment by sadko4u** [comment]
Oh, it seems to be reproducible with large input buffer size of 8192 samples.

**Comment by LebedevRI** [comment]
Yup, that's the buffer size i seem to have.

**Comment by sadko4u** [comment]
Yes, that was silly mistake, here's the fix:
https://github.com/lsp-plugins/lsp-plugins-comp-delay/commit/78dde4177a6c9cb818d3c04d10da6affa4aedfb1

**Comment by LebedevRI** [comment]
(Reported to debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1054194)

**Comment by sadko4u** [comment]
Fixed in 1.2.13
